corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -89,11 +89,7 @@
                 $loading = $main.find(".hq-loading").remove(),
                 $loadingError = $main.find(".hq-loading-error").remove();
 
-            $('#demand-releases').on('show.bs.tab', _.throttle(function (event) {
-                if (event && (event.metaKey || event.ctrlKey || event.which === 2)) {
-                    window.open("{% url 'release_manager' domain app.get_id %}", '_blank');
```

Comment:
this was never running, because the event here is the show.bs.tab, not the click, so metaKey, etc. are never set